src/Utils/String.idr

```diff
@@ -11,3 +11,8 @@ dotSep (x :: xs) = x ++ concat ["." ++ y | y <- xs]
 export
 stripQuotes : (str : String) -> String
 stripQuotes str = substr 1 (length str `minus` 2) str
+
+export
+lowerFirst : String -> Bool
```

Comment:
There is a duplicate definition of this function at [src/TTImp/Utils.idr](https://github.com/idris-lang/Idris2/blob/d105dd11a7e76dae28c34a43f931608b0e08c95e/src/TTImp/Utils.idr#L12)

Reply:
Nice catch!